lib/chai/utils/inspect.js

```diff
@@ -230,6 +241,28 @@ function formatArray(ctx, value, recurseTimes, visibleKeys, keys) {
   return output;
 }
 
+function formatTypedArray(ctx, value, recurseTimes, visibleKeys, keys) {
+  var output = [];
+
+  // Truncating TypedArray if it has more than 1000 elements
+  var l = value.length > 1000 ? 5 : value.length;
+
+  for (var i = 0; i < l; ++i) {
+    if (Object.prototype.hasOwnProperty.call(value, String(i))) {
+      output.push(formatProperty(ctx, value, recurseTimes, visibleKeys,
+          String(i), true));
```

Comment:
To be honest I haven't used Typed Arrays at all so don't know the best way to show two in a comparison - is showing the start a good idea? For files expressed as typed arrays the first few bytes are going to be magic bytes right, so will largely always be identical?